CRM/Core/DAO.php

```diff
@@ -1079,16 +1079,16 @@ public static function checkFieldExists($tableName, $columnName, $i18nRewrite =
    *
    * @return array
    */
-  public static function getTableNames() {
+  public static function getTableNames(): array {
```

Comment:
I think a followup could be to just double-confirm that this function and the associated UF output is meaningless now if using a recent version of civicrm_entity which figures out the single/separate db on its own. This function isn't used anywhere else in universe.

I like the docblock comment though.